staging/src/k8s.io/sample-apiserver/artifacts/example/rbac.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sample-apiserver-clusterrole
```

Comment:
I wonder whether the role should have a generic name, to be re-used by all aggregated apiservers. aggregated-apiserver-clusterrole?

Reply:
That one sounds good as well. Updated.